.github/workflows/release-docker.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 env:
   DOCKER_CLI_EXPERIMENTAL: enabled
-  IMAGE_NAME: decred/dcrdex
+  IMAGE_NAME: decred/bisonwallet
```

Comment:
Can you comment on this change?

Reply:
This is the name of the docker image.  Currently only referenced in the Umbrel app manifest; the reason of the change is to be able to use the BW name in the umbrel app store listing PR.